app/views/conservation_records/_form.html.erb

```diff
@@ -14,17 +14,17 @@
 
   <div class="form-block">
     <div class="form-group">
-      <%= form.label :date_received_in_preservation_services %>
+      <%= form.label :date_received_in_preservation_services %> <span style="color: #e00122;">*</span>
       <%= form.date_field :date_received_in_preservation_services, class: 'form-control' %>
     </div>
 
     <div class="form-group">
-      <%= form.label :department %>
+      <%= form.label :department %> <span style="color: #e00122;">*</span>
       <%= form.select(:department, options_from_collection_for_select(@departments, "id", "key", @conservation_record.department), { prompt: 'Select Department' }, {class: 'form-control'}) %>
     </div>
 
   <div class="form-group">
-    <%= form.label :title %>
+    <%= form.label :title %> <span style="color: #e00122;">*</span>
     <%= form.text_field :title, class: 'form-control' %>
```

Comment:
For required fields, can you also add a `required: true` to the end?  Something like `<%= form.text_field :title, class: 'form-control', required: true %>`

It's considered good practice.  It catches the empty field before the user submits and I think it helps with accessibility by adding some markup that helps people with screen readers detect required fields.

Reply:
@crowesn I don't see that you've added `required: true` like I suggested.  Any reason for not adding that to required fields?

Reply:
apologies, I thought I had read this suggestion but then I didn't see it on a second reading. I will add.